Mu2eG4/geom/extmon_fnal_v02.txt

```diff
@@ -32,7 +32,21 @@ string extMonFNAL.concreteMaterial = "ShieldingConcrete";
 // may be different):  {upstream size, downstream size}
 vector<double> extMonFNAL.collimator1.channelDiameter  = {50., 50.};
 vector<double> extMonFNAL.collimator1.alignmentPlugDiameter = { 219.1, 355.6 };
-vector<double> extMonFNAL.collimator1.alignmentHoleDiameter = { 247.7, 381.0 };
+vector<double> extMonFNAL.collimator1.alignmentPlugInnerShellThickness = { 5.0, 5.0 };
+vector<double> extMonFNAL.collimator1.alignmentPlugOuterShellThickness = { 8.2, 11.1 };
+
+// Shot Liner Values
+vector<double> extMonFNAL.collimator1.shotLinerInnerDiameter = { 247.7, 381.0 };
+vector<double> extMonFNAL.collimator1.shotLinerInnerThickness = { 12.7, 12.7 };
+double extMonFNAL.collimator1.shotLinerOuterDiameter = 609.6;
+double extMonFNAL.collimator1.shotLinerOuterThickness = 9.7;
+double extMonFNAL.collimator1.length = 4203.7;
+
+// Steel Shot material
+// Density calcuated by multiplying the density of 'normal steel' (7.85g/cm^3)
+// by the densest packing of spheres (63.5%)
+string         extMonFNAL.collimator1.shot.materialName  = "SteelShot";
```

Comment:
I suggest using a more descriptive name: ClosePackedExtMonSteelShot . 

Also, I would not write the numerical values here - if you ever update the numbers in the .cc file it's hard to remember to fix these.  Although that's not likely, don't create multiple points of maintenance unless you are forced to by external constraints.

Same comment for collimator2.shot.materialName.

Reply:
Thanks.